### ISSUE TITLE
void-function frame-focus-state in Emacs 26.3

### ISSUE THREAD

**Issue by bcclaywell** [emacs-lsp/lsp-ui#391]
I'm on Emacs 26.3. Presumably after #381 I'm now getting

```
Error in post-command-hook (lsp-ui-doc--glance-hide-frame): (void-function frame-focus-state)
```

when the doc popup tries to go away. frame-focus-state doesn't appear to exist in frame.el for Emacs 26.

**Comment by yyoncho** [comment]
@kiennq would you take a look? Seems like `frame-focus-state` is only in emacs27.

**Comment by kiennq** [comment]
Sorry, it should have been fixed with #392
